docs/create-packages/creating-a-package-msbuild.md

```diff
@@ -86,7 +86,7 @@ If you are using MSBuild with a non-SDK-style project and PackageReference, add
    ```xml
    <ItemGroup>
      <!-- ... -->
-     <PackageReference Include="NuGet.Build.Tasks.Pack" Version="5.2.0"/>
+     <PackageReference Include="NuGet.Build.Tasks.Pack" Version="6.7.0" PrivateAssets="all" Pack="false" />
```

Comment:
I don't think you need `Pack="false"`. I'm not really sure what it does anyways. 

It's not really a supported PackageReference metadata. 
https://learn.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files

Also, maybe the comment on 88 can point to https://learn.microsoft.com/en-us/nuget/release-notes/ (via a relative link), and pointing out to reference the *latest* package version.

Reply:
`Pack="false"` is read by items that correspond to files, and therefore can be a way to prevent that file from being included in the package (although from memory it ALSO needs `PackagePath="??" metadata). But for PackageReference, it has no effect.

Reply:
> But for PackageReference, it has no effect.

That's my understanding too. I've just seen it...a lot.

Reply:
Remove pack suggestion

```suggestion
     <PackageReference Include="NuGet.Build.Tasks.Pack" Version="6.7.0" PrivateAssets="all" />
```